STOL/flightstate.py

```diff
@@ -4,8 +4,9 @@ class FlightState(Model):
     def setup(self):
 
         rho = Variable("\\rho", 1.225, "kg/m**3", "air density")
+        mu = Variable("\\mu", 1.789e-5, "kg/m/s", "air viscosity")
         V = Variable("V", "knots", "speed")
 
-        constraints = [rho == rho, V == V]
+        constraints = [rho == rho, V == V, mu == mu]
```

Comment:
this whole constraints list should be unnecessary on master!

Reply:
yeah sorry, i need to delete this